yarn.lock

```diff
@@ -1746,11 +1746,6 @@ is-reference@^3.0.0:
   dependencies:
     "@types/estree" "*"
 
-is-unicode-supported@^1.3.0:
```

Comment:
Why is the yarn.lock file changed? I think this PR doesn't add any library so it shouldn't trigger a change in the file.

Reply:
If you switch to master branch and run `yarn`, this one will also be removed too. 🤔️